[2025-04-06T02:39:17.184+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-06T02:39:17.267+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_store_data.trigger_dbt_run scheduled__2025-04-05T00:00:00+00:00 [queued]>
[2025-04-06T02:39:17.308+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_store_data.trigger_dbt_run scheduled__2025-04-05T00:00:00+00:00 [queued]>
[2025-04-06T02:39:17.309+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-06T02:39:17.401+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): trigger_dbt_run> on 2025-04-05 00:00:00+00:00
[2025-04-06T02:39:17.407+0000] {standard_task_runner.py:72} INFO - Started process 177 to run task
[2025-04-06T02:39:17.412+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_and_store_data', 'trigger_dbt_run', 'scheduled__2025-04-05T00:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/reverse_etl_dag.py', '--cfg-path', '/tmp/tmppvrej1f1']
[2025-04-06T02:39:17.416+0000] {standard_task_runner.py:105} INFO - Job 80: Subtask trigger_dbt_run
[2025-04-06T02:39:17.582+0000] {task_command.py:467} INFO - Running <TaskInstance: fetch_and_store_data.trigger_dbt_run scheduled__2025-04-05T00:00:00+00:00 [running]> on host 830eaf47e7f8
[2025-04-06T02:39:17.813+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fetch_and_store_data' AIRFLOW_CTX_TASK_ID='trigger_dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-05T00:00:00+00:00'
[2025-04-06T02:39:17.815+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-06T02:39:17.837+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-06T02:39:17.839+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', ' cd /usr/local/airflow/dags/reverse_etl && dbt run']
[2025-04-06T02:39:17.853+0000] {subprocess.py:99} INFO - Output:
[2025-04-06T02:39:23.863+0000] {subprocess.py:106} INFO - [0m02:39:23  Running with dbt=1.9.4
[2025-04-06T02:39:27.220+0000] {subprocess.py:106} INFO - [0m02:39:27  Registered adapter: postgres=1.9.0
[2025-04-06T02:39:30.354+0000] {subprocess.py:106} INFO - [0m02:39:30  Found 1 model, 1 source, 434 macros
[2025-04-06T02:39:30.374+0000] {subprocess.py:106} INFO - [0m02:39:30
[2025-04-06T02:39:30.377+0000] {subprocess.py:106} INFO - [0m02:39:30  Concurrency: 2 threads (target='dev')
[2025-04-06T02:39:30.381+0000] {subprocess.py:106} INFO - [0m02:39:30
[2025-04-06T02:39:41.823+0000] {subprocess.py:106} INFO - [0m02:39:41  1 of 1 START sql table model aggregated.survival_rate .......................... [RUN]
[2025-04-06T02:39:46.365+0000] {subprocess.py:106} INFO - [0m02:39:46  1 of 1 OK created sql table model aggregated.survival_rate ..................... [[32mSELECT 0[0m in 4.54s]
[2025-04-06T02:39:49.556+0000] {subprocess.py:106} INFO - [0m02:39:49
[2025-04-06T02:39:49.557+0000] {subprocess.py:106} INFO - [0m02:39:49  Finished running 1 table model in 0 hours 0 minutes and 19.17 seconds (19.17s).
[2025-04-06T02:39:49.680+0000] {subprocess.py:106} INFO - [0m02:39:49
[2025-04-06T02:39:49.682+0000] {subprocess.py:106} INFO - [0m02:39:49  [32mCompleted successfully[0m
[2025-04-06T02:39:49.683+0000] {subprocess.py:106} INFO - [0m02:39:49
[2025-04-06T02:39:49.686+0000] {subprocess.py:106} INFO - [0m02:39:49  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-04-06T02:39:51.712+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-06T02:39:51.865+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-06T02:39:51.867+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fetch_and_store_data, task_id=trigger_dbt_run, run_id=scheduled__2025-04-05T00:00:00+00:00, execution_date=20250405T000000, start_date=20250406T023917, end_date=20250406T023951
[2025-04-06T02:39:52.074+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-06T02:39:52.110+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-06T02:39:52.114+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
